CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 # History
 
+## 3.2.0
+- Widget:
+    - `UploadcareWidget.selectFile()` was deprecated since `Fragment.startActivityForResult` was deprecated.
```

Comment:
Maybe like this?

```suggestion
    - `UploadcareWidget.selectFile()` and `Fragment.startActivityForResult` were deprecated.
```

Reply:
The methods `UploadcareWidget.selectFile()` and `Fragment.startActivityForResult` have been deprecated.

Reply:
@onthecrow WDYT?

Reply:
@rsedykh Sounds fine for me